typescript/src/scss/main.scss

```diff
@@ -6,11 +11,12 @@
     margin: 20px 0;
 
     td, th {
-      text-align: right;
+      text-align: center;
```

Comment:
We should probably only centre align `th` not `td`, as we want the numbers to be right-aligned.

Reply:
Fixed that now

Reply:
fixed on line 25